src/Paket.Core/PaketConfigFiles/ProjectFile.fs

```diff
@@ -1671,6 +1671,22 @@ module ProjectFile =
               Kind = NugetPackageKind.DotnetCliTool
               TargetFramework = None })
 
+    let getAutoGenerateBindingRedirects (project:ProjectFile) = getProperty "AutoGenerateBindingRedirects" project
+    let setOrCreateAutoGenerateBindingRedirects (project:ProjectFile) =
+        if (getAutoGenerateBindingRedirects project).IsSome then
+            project.Document
+            |> getDescendants "AutoGenerateBindingRedirects"
+            |> List.iter(fun x -> x.InnerText <- "true")
+        else
+            let propertyGroups = project.Document |> getDescendants "PropertyGroup"
+            if propertyGroups.IsEmpty |> not then
```

Comment:
please change to pattern matching

`match propertyGroups with
| x :: _ -> x.AppendChild (createNodeSet "AutoGenerateBindingRedirects" "true" project) |> ignore
| _ -> ()`